mapviz_plugins/include/mapviz_plugins/measuring_plugin.h

```diff
@@ -31,6 +31,7 @@
 #define MAPVIZ_PLUGINS_MEASURING_PLUGIN_H_
 
 #include <mapviz/mapviz_plugin.h>
+#include <mapviz_plugins/point_drawing_plugin.h>
```

Comment:
Why is this being included here?

Reply:
It shouldn't be. Should I remove and re-commit?

Reply:
Yeah, please do.